pymedphys/_dicom/anonymise/__init__.py

```diff
@@ -64,7 +65,47 @@ def create_filename_from_dataset(ds, dirpath=""):
     return pjoin(dirpath, "{}.{}.dcm".format(mode_prefix, ds.SOPInstanceUID))
 
 
-def anonymise_dataset(  # pylint: disable = inconsistent-return-statements
+def anonymise_dataset(ds, copy_dataset=True, replacement_strategy=None):
```

Comment:
```suggestion
def anonymise_dataset(ds, copy=True, strategy=None):
```

Also, can we potentially simplify the API down to just "strategy"?

Reply:
If you want to modify DicomBase and test_base, yes.
This was left in/put back in to avoid having to dig down further in to other "unrelated" code.